README.md

```diff
@@ -24,7 +24,7 @@ cd OData
 build
 ```
 ### Testing
-Each solution contains some test projects. Test projects use xUnit runner nuget package. Open the project, build it, and then test cases should appear in test explorer. Run all the tests in the test explorer. For running end-to-end tests you must open the solution as *Administrator*. More detail at [this](http://odata.github.io/WebApi/#09-01-unittest-e2etest).
+Each solution contains some test projects. Test projects use xUnit runner nuget package. Open the project, build it, and then test cases should appear in test explorer. If not, this is because the assemblies are delay signed and you're missing the private key so xunit will not load them in Visual Studio. To fix, un-check "Sign the assembly" in all project properties and remove the "PublicKey=" portion of all InternalsVisibleTo assembly attributes (do not check in this temporary workaround). Run all the tests in the test explorer. For running end-to-end tests you must open the solution as *Administrator*. More detail at [this](http://odata.github.io/WebApi/#09-01-unittest-e2etest).
 
```

Comment:
Yes it will, and that works for the command line, but it doesn’t work for visual studio.  At least not for me.

From: Congyong Su [mailto:notifications@github.com]
Sent: Monday, December 14, 2015 6:58 PM
To: OData/WebApi <WebApi@noreply.github.com>
Cc: Nick Kinnan <nickkin@microsoft.com>
Subject: Re: [WebApi] Added config options "SerializeNullCollectionsAsEmpty" and "DoNotSeri… (#583)


In README.md<https://github.com/OData/WebApi/pull/583#discussion_r47591657>:

> @@ -24,7 +24,7 @@ cd OData

>  build

>  ```

>  ### Testing

> -Each solution contains some test projects. Test projects use xUnit runner nuget package. Open the project, build it, and then test cases should appear in test explorer. Run all the tests in the test explorer. For running end-to-end tests you must open the solution as *Administrator*. More detail at [this](http://odata.github.io/WebApi/#09-01-unittest-e2etest).

> +Each solution contains some test projects. Test projects use xUnit runner nuget package. Open the project, build it, and then test cases should appear in test explorer. If not, this is because the assemblies are delay signed and you're missing the private key so xunit will not load them in Visual Studio. To fix, un-check "Sign the assembly" in all project properties and remove the "PublicKey=" portion of all InternalsVisibleTo assembly attributes (do not check in this temporary workaround). Run all the tests in the test explorer. For running end-to-end tests you must open the solution as *Administrator*. More detail at [this](http://odata.github.io/WebApi/#09-01-unittest-e2etest).

>

You don't need this. It will automatically prompt you if you run the build.cmd:

C:\repo\OData\WebApi\OData>build

Microsoft (R) Build Engine version 12.0.40629.0

[Microsoft .NET Framework, version 4.0.30319.34014]

Copyright (C) Microsoft Corporation. All rights reserved.



C:\repo\OData\WebApi\OData\WebApiOData.msbuild(113,5): error : Unit tests will not run correctly unless SkipStrongNames

 is Enabled. Current status: Disabled. Run build.cmd EnableSkipStrongNames to fix this problem.

—
Reply to this email directly or view it on GitHub<https://github.com/OData/WebApi/pull/583/files#r47591657>.